types/superagent/index.d.ts

```diff
@@ -67,13 +67,20 @@ declare namespace request {
         unsubscribe(url: string, callback?: CallbackHandler): Req;
     }
 
+    interface ErrorResponse extends Error {
```

Comment:
Sure thing, changed it: https://github.com/SamuelMarks/DefinitelyTyped/commit/0fe027fe89391de491d7002adda078439d7a2956 on my [superagent-patch-error](https://github.com/SamuelMarks/DefinitelyTyped/tree/superagent-patch-error) branch.

(not sure why it didn't get appended to this PR?)